crates/bevy_ecs/src/lib.rs

```diff
@@ -48,6 +48,7 @@ pub mod prelude {
         },
         world::{FromWorld, World},
     };
+    pub use bevy_ecs_macros::States;
```

Comment:
Where are the derives for Component and Resource imported?

Reply:
They are re-exported in their respective source files -- `component.rs` and `system_param.rs`. You usually want to re-export a derive macro into the same scope as its trait, that way `use bevy::ecs::schedule::States` will import both the trait and the derive macro.